info/token-info-storage-bit.json

```diff
@@ -112,5 +112,34 @@
             "en":"EN text about another token",
             "de":"DE text about another token"
         }             
+    },
+	    {
+        "token_id":"e2fa409b9150b6489ec9913d3afbc7db1fe0e94ac09537e458aea673d341367a",
```

Comment:
This token does not exist in the BIT network. But this change is for BIT tokens.